### PR TITLE
Fix bug that shoots the player back when tongue is pointing down

### PR DESCRIPTION
**Pull request by fheysel**
Update so grapple direction is more accurate. Solves bug described in FGTS-152

### CODE REVIEW

File: godotproject/Player.tscn [hunk 1]
```diff
@@ -281,11 +281,8 @@ blend_point_3/node = SubResource( 23 )
 blend_point_3/pos = Vector2( 0, -1 )
 blend_mode = 1
 
-[sub_resource type="AnimationNodeAnimation" id=83]
-animation = "Hurt"
-
 [sub_resource type="AnimationNodeAnimation" id=25]
-animation = "Idle_froggie"
+animation = "Hurt"
```
Comment (charasyn) [inline godotproject/Player.tscn:285]: Not asking for any change to the PR here, but just.. wtf is going on in this file??? Godot decided to just randomly change the IDs of all the animations or smth?

File: godotproject/Player.gd [hunk 2]
```diff
@@ -435,20 +435,19 @@ func _physics_process(delta):
...
 			stop_swing()
 			
-			# A unit vector in the direction of the mouth to the tip of tongue.
-			# It will be used to calculate the launch direction.
-			# The direction is calculated as the vector perpendicular to the current velocity,
-			# we divide by the magnitude to make it a unit vecotr. This helps to normalize it.
-			var tongue_direction = Vector2(velocity.y, -1*velocity.x).normalized()
+			var frog_pos = get_global_transform().xform(Vector2.ZERO)
+			var swing_vec = swing_pivot_position - frog_pos 
+			var swing_vec_norm = swing_vec.normalized()
```
Comment (charasyn) [inline godotproject/Player.gd]: Could you possibly rename this to `tongue_vec`? I don't want it to be confused with the `swing_vec` used earlier on in the file, which is the opposite vector (from swing position to frog)

**Review by charasyn — CHANGES_REQUESTED** [review]
One bit of feedback about renaming the variables, other than that looks good!

File: godotproject/Player.tscn (hunk 1 above)
Comment (fheysel) [reply godotproject/Player.tscn:285]: yeah idk, anytime I update a .gd file, godot always does some weird stuff with the tscn file. It still runs fine so I generally just ignore it

File: godotproject/Player.gd (hunk 2 above)
Comment (fheysel) [reply godotproject/Player.gd]: sure

**Review by charasyn — APPROVED** [review]
Thanks! Looks good

**Pull request merged**